types/api.yaml

```diff
@@ -0,0 +1,53 @@
+# API specific types
+
+ValidatorResponse:
+  type: object
+  properties:
+    index:
+      allOf:
+        - $ref: './primitive.yaml#/schemas/Uint64'
+        - description: "Index of validator in validator registry."
+    balance:
+      allOf:
+        - $ref: "./primitive.yaml#/schemas/Gwei"
+        - description: "Current validator balance in gwei."
+    status:
+      $ref: "#/ValidatorStatus"
+    validator:
+      $ref: "./validator.yaml#/Validator"
+
+
+ValidatorStatus:
+  description: |
+    Possible statuses:
```

Comment:
Can I just specify a primary status (e.g. `pending`) rather than a full status with substatus (e.g. `pending_initialized`)?

If so, we should note that

Reply:
Ah, i see  that `enum` does not list primary statuses individually. 

It might be useful to be able to just say `active` and get all the active's without having to list all active sub-statuses

Reply:
So in response, we have these details statuses, which in query param are extended with "active". Yeah we have inheritance here :rofl:

Reply:
https://github.com/ethereum/eth2.0-APIs/pull/44/files#diff-564ddc01cfd0c4c456b0e2b545375f0eR31